project/Publish.scala

```diff
@@ -40,21 +40,16 @@ object Publish extends AutoPlugin {
   }
 
   private def akkaPublishTo = Def.setting {
-    sonatypeRepo(version.value).orElse(localRepo(defaultPublishTo.value))
+    val key = new java.io.File(
+      Option(System.getProperty("akka.gustav.key")).getOrElse(System.getProperty("user.home") + "/.ssh/id_rsa_gustav.pem"))
```

Comment:
one more thing, update release setup instructions with information that id_rsa_gustav.pem must exist
(is missing creds caught by a dry-run?)

Reply:
I updated the setup instructions.

It is only needed for publishing snapshots, not when publishing releases, so it doesn't have to be part of the dry-run (yet).

Reply:
right, my thinko